frontend/src/components/maps/ParcelPopupView.scss

```diff
@@ -4,27 +4,37 @@
   max-width: 250px;
   box-shadow: 1px 2px 2px;
   text-align: left;
-  padding: 12px;
+  padding: 10px;
   .row {
     flex-direction: column;
   }
   .row .col :first-child {
     margin-top: 0px;
   }
+  .pid {
+    text-align: center;
+  }
+  .name {
+    font-weight: bold;
+  }
 
   .label {
     font-weight: 700;
+    padding-right: 5px;
   }
 
   .list-group {
     margin-top: 6px;
   }
   .list-group-item {
-    padding: 0;
+    padding: 0px 0px 5px 0px !important;
```

Comment:
Uggh, another `important!`.  Styling is a mess...

Reply:
Same comment as for the building popups.